package.json

```diff
@@ -19,14 +19,14 @@
   },
   "packageManager": "yarn@2.0.0-rc.29",
```

Comment:
Could you update this version as well please, otherwise corepack will spawn an old version of v2 which fails on parsing newer settings in `~/.yarnrc.yml`